context/app/static/js/pages/Donor/Donor.jsx

```diff
@@ -8,28 +8,31 @@ import Protocol from 'js/components/detailPage/Protocol';
 import DetailLayout from 'js/components/detailPage/DetailLayout';
 import useSendUUIDEvent from 'js/components/detailPage/useSendUUIDEvent';
 import useEntityStore from 'js/stores/useEntityStore';
+
 import DetailContext from 'js/components/detailPage/context';
 import { getSectionOrder } from 'js/components/detailPage/utils';
 import DerivedEntitiesSection from 'js/components/detailPage/derivedEntities/DerivedEntitiesSection';
 
 const entityStoreSelector = (state) => state.setAssayMetadata;
 
-function DonorDetail() {
+function DonorDetail({ assayMetadata }) {
   const {
-    entity: {
-      uuid,
-      protocol_url,
-      hubmap_id,
-      entity_type,
-      mapped_metadata = {},
-      created_timestamp,
-      last_modified_timestamp,
-      description,
-      group_name,
-      created_by_user_displayname,
-      created_by_user_email,
-    },
-  } = useFlaskDataContext();
+    uuid,
+    protocol_url,
+    group_name,
+    created_by_user_displayname,
+    created_by_user_email,
+    hubmap_id,
+    entity_type,
+    created_timestamp,
+    last_modified_timestamp,
+    description,
+    mapped_metadata = {},
+    // As data comes in from other consortia, we won't be able
+    // to rely on donor metadata always being available.
+    // Unpublished HuBMAP data may also be missing donor metadata.
+  } = assayMetadata;
```

Comment:
This could be a potential source of the issues we're seeing - this was getting deconstructed from assayMetadata before the changes to `useFlaskDataContext`, and was being deconstructed from `entity` afterwards; let me try changing this on main to see if it's a potential culprit so we don't have to roll back/redo as many changes

Reply:
Never mind, that wasn't it 👎